package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "bs4-breakpoint-check",
-  "version": "1.0.5",
+  "version": "1.1.5",
```

Comment:
@iflorespaz can you please set next version to `1.1.0`? thanks!

Reply:
Done.